twiml/message/message/message-4/message-4.3.x.js

```diff
@@ -1,7 +1,4 @@
 const MessagingResponse = require('twilio').twiml.MessagingResponse;
 
 const response = new MessagingResponse();
-const message = response.message();
 message.media('https://demo.twilio.com/owl.png');
-
-console.log(response.toString());
```

Comment:
@juank-pa not sure if this change is ok, could you please check that this is equal to master.